### DISCUSSION TITLE
Use Case: Postage Stamp Management

### DISCUSSION THREAD

**Discussion by russell-hutson-jp** [inventree/InvenTree#6529] [Q&A]
Hi

I am assessing InvenTree for use in a Postal Operator.
* We will purchase new Stamps from a few Suppliers
* We will receive them into our "warehouse" - actually a safe in our main post office
* Branch offices may order stock from the main post office
* We need to track stock being picked from the Safe. This happens once a week and consolidates all that week's branch office orders
* Branch Offices need to confirm receipt

So I'm happy with setting up Suppliers and raising Purchase orders in InvenTree.
I have set up the Main Post Office as a Location and the Safe as a Sub-location.

I'm not sure if I should treat Branch Offices as Customers and have them raise Sales Orders (can I set up access to allow that?).
Or do I use locations? If so, how can a Branch Office ask for more supplies of Stamps?

Thoughts appreciated. Thanks in advance

**Comment by SergeoLacruz** [comment]
New use case :-) nice
If a Branch Office orders stamps at your office you can create a sales order and track the stock with that. 
If I understand you right you want your customers to order via InvenTree. I think this is work for something like an CRM tool (customer relation management). I am not sure if InvenTree is the right tool for that. 
Michael

**Reply by russell-hutson-jp** [reply]
Thanks. Our existing system/process relies on the Branch ordering more stamps using a spreadsheet/email.
Our central office then manually enters orders into the system. So we can keep the same process for now, but using InvenTree.

**Comment by matmair** [comment]
Hi there! We support Role-based access control so you can give certain groups of users limited access rights. If you only need to track the branches it went to you could use customers for that; if you want to have the option to track further I would suggest locations.

Not sure if core is fully set up for what you need, feel free to reach out to support@inventree.org (maintainer) / code@mjmair.com (me) if you need adoptions for your specific use case.

**Reply by russell-hutson-jp** [reply]
Thanks. I have set up each branch as a Customer for now. Unfortunately, user Groups can't be locked to a Customer from what I can in Inventree. So I'm happy to accept that Inventree should not be customer facing.

I think what you suggest will work for us if I remove the requirement to allow customers to raise sales orders. 

Perhaps later I can add a Customer facing UI using the API.
